README.md

```diff
@@ -94,6 +94,21 @@ Available query commands:
   - Options:
     - `--rtl`: Display fields in right-to-left order
 
+### Updating Vector Store Content
+
+To update or add content to the vector store:
+
+```bash
+# Update all contexts for the takanon bot (complete rebuild)
+botnim sync staging takanon --backend es --replace-context
```

Comment:
would have been much better to have `--replace-context` to accept a parameter, so that `--replace-context all` replaces all contexts and `--replace-context context` replaces one context.
This is instead of propagating another parameter down the line and creating conflicting states ('--context-to-update' with '--replace-context' together)